components/google_dialogflow/actions/list-entity-types/list-entity-types.mjs

```diff
@@ -0,0 +1,27 @@
+import googleDialogflow from "../../google_dialogflow.app.mjs";
+
+export default {
+  type: "action",
+  key: "google_dialogflow-list-entity-types",
+  version: "0.0.1",
+  name: "List Entity Types",
+  description: "Retrieves list of entity types, [See REST docs](https://cloud.google.com/dialogflow/es/docs/reference/rest/v2/projects.agent.entityTypes/list) and [client API docs](https://googleapis.dev/nodejs/dialogflow/latest/google.cloud.dialogflow.v2beta1.EntityTypes.html#listEntityTypes2)",
+  props: {
+    googleDialogflow,
+    languageCode: {
+      propDefinition: [
+        googleDialogflow,
+        "languageCode",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const response = await this.googleDialogflow.listEntityTypes({
+      languageCode: this.languageCode,
+      pageSize: 5,
+      pageToken: null,
+    });
+    $.export("$summary", `${response[0].length} entity types has been retrieved.`);
```

Comment:
Can you check for no records and add a specific message for it?

Reply:
Done